catalog-spec/catalog-spec.md

```diff
@@ -150,6 +186,7 @@ A typical '_child_' sub-catalog could look similar:
 ```json
 {
   "stac_version": "0.7.0",
+  "stac_extensions": ["eo"],
```

Comment:
I had in my notes, that we were going to add versions to the extensions.
Something like this?
```
{
"custom_extension_1": "https://....",
"eo": "0.0.1",
"cube": "0.0.1"
}
```

Reply:
See my comment below. I'm pretty sure we settled on arrays as the version information is basically included in the URL as the referenced schema is versioned.